src/main.ts

```diff
@@ -29,25 +30,23 @@ async function bootstrap() {
   app.enable('trust proxy');
   app.useLogger(logger);
   app.enableCors();
-  app.setGlobalPrefix('api/v1', { exclude: ['/', 'health', 'api', 'api/v1', 'api/docs'] });
+  app.setGlobalPrefix('api/v1', { exclude: ['/', 'health'] });
+  app.useGlobalInterceptors(new ResponseInterceptor());
 
   // TODO: set options for swagger docs
   const options = new DocumentBuilder()
-    .setTitle('<project-title-here>')
+    .setTitle('Remote Bingo')
     .setDescription('<project-description-here>')
     .setVersion('1.0')
     .addBearerAuth()
     .build();
 
   const document = SwaggerModule.createDocument(app, options);
-  SwaggerModule.setup('api/docs', app, document);
+  SwaggerModule.setup('api', app, document);
```

Comment:
Why did you change this again? Was this at least approved by the team lead? 
cc: @Heba-WebDev

Reply:
Someone overwrote that file and i blindly reverted to an older commit. I'm using the current one now

Reply:
I think it was decided that it would be api/docs

Reply:
Yes